src/components/_modals/BuyOrSellModal.tsx

```diff
@@ -39,75 +43,82 @@ export const BuyOrSellModal = ({
         >
           Available on
         </Text>
-        {uniswapLink && (
-          <Link
-            href={uniswapLink}
-            onClick={() => {
-              analytics.track("strategy.buy-sell", {
-                strategyCard: cellarDataMap[id].name,
-                platformSelection: "Uniswap",
-                landingType: landingType(),
-              })
-            }}
-            target="_blank"
-            style={{ textDecoration: "none" }}
-          >
-            <HStack
-              justifyContent="space-between"
-              backgroundColor="surface.secondary"
-              padding={4}
-              borderRadius="xl"
-              _hover={{
-                backgroundColor: "purple.dark",
+        {exchange?.map((value) => {
+          if (!value.url) {
+            return (
+              <Link
+                href={`/strategies/${id}/manage`}
+                onClick={() => {
+                  analytics.track("strategy.buy-sell", {
+                    strategyCard: cellarDataMap[id].name,
+                    platformSelection: `${value.name}`,
+                    landingType: landingType(),
+                  })
+                }}
+                style={{ textDecoration: "none" }}
```

Comment:
Was the usual `textDecoration` prop not working for some reason? 👀 

I see this line was pre-existing to this PR, just wondering if the style prop is necessary